### PR TITLE
Update messaging for protected pages

### PR DESCRIPTION
**Pull request by Simon-Busch**
⚠️ jut add date and link before merging

### CODE REVIEW

**Comment by Simon-Busch** [comment]
@captainmangoC4 Let me know this is OK for you :)

File: src/components/Modal.js [hunk 1]
```diff
@@ -23,27 +23,29 @@ const Modal = ({
...
+              className="button cta-button primary"
+              onClick={primaryButtonAction}
+            >
+              {primaryButtonText || "Ok"}
+            </button>
+          </div>
+        ) : (
+          ""
+        )}
       </div>
```
Comment (captainmangoC4) [inline src/components/Modal.js:49]: I think you can undo the changes to this file. We won't have any modals without action buttons.

File: src/components/Login/Login.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Dropdown from "../Dropdown";
 import * as styles from "./Login.module.scss";
 import * as dropdownStyles from "../Dropdown.module.scss";
 
-const Login = () => {
+const Login = ({ dropDownViewOff }) => {
```
Comment (captainmangoC4) [inline src/components/Login/Login.tsx:13]: I would give this a name that reflects what it does show, vs what it doesn't show and then set it to false by default. For example: 
```javascript
const Login = ({ displayAsButtons = false }) => {
```
If we don't give it a default value, or define an interface for props where this is optional, then we will have typescript errors in the consuming components if they don't pass this prop.

File: src/components/reporter/SubmitFindings.js [hunk 3]
```diff
@@ -438,7 +440,18 @@ const SubmitFindings = ({ wardensList, sponsor, contest, repo }) => {
       )}
     </div>
   ) : (
-    <p>You must connect your wallet to submit findings</p>
+    <div className="login-container">
+      <div className="finding-error-container">
+        <h1>Please login</h1>
+        <p>
```
Comment (captainmangoC4) [inline src/components/reporter/SubmitFindings.js:446]: Let's add a todo comment here to remind us that we need to include the official date once we know when this will be released

File: src/styles/forms.css [hunk 4]
```diff
@@ -211,3 +211,25 @@ label {
...
+  width: 50vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--secondary-background-purple);
+  border-radius: 8px;
+  border:1px solid var(--header-background-purple);
+  padding:20px;
+  box-shadow: 0px 4px 10px rgba(0,0,0,0.5);
+}
```
Comment (captainmangoC4) [inline src/styles/forms.css:235]: Is it possible to just utilize the className and styles that are applied to the form? e.g. from the Form module, `Form`

**Review by captainmangoC4 — CHANGES_REQUESTED** [review]

File: src/styles/forms.css (hunk 4 above)
Comment (captainmangoC4) [reply src/styles/forms.css:235]: You can also apply the `centered-text` class name

File: src/components/reporter/SubmitFindings.js (hunk 3 above)
Comment (captainmangoC4) [reply src/components/reporter/SubmitFindings.js:446]: I see your note in the PR comment, but I think it's a good idea to add a comment in the code since this PR will be closed before the full feature is released.

File: src/components/Modal.js (hunk 1 above)
Comment (Simon-Busch) [reply src/components/Modal.js:49]: Done :-)

File: src/components/Login/Login.tsx (hunk 2 above)
Comment (Simon-Busch) [reply src/components/Login/Login.tsx:13]: Totally make sense ! I adapted it as well ✅

File: src/components/reporter/SubmitFindings.js (hunk 3 above)
Comment (Simon-Busch) [reply src/components/reporter/SubmitFindings.js:446]: Yes correct so we make sure we change it before warden auth merge, Done ✅

File: src/styles/forms.css (hunk 4 above)
Comment (Simon-Busch) [reply src/styles/forms.css:235]: Totally, Doesn't make sense to have duplicate classes, I changed it :) ✅

File: src/components/Dropdown.module.scss [hunk 5]
```diff
@@ -38,7 +38,6 @@
 .Button {
   @include button;
   background: var(--dark-purple);
-  width: 100%;
```
Comment (captainmangoC4) [inline src/components/Dropdown.module.scss:41]: This is actually necessary for the buttons in the dropdown. Otherwise they look like this:
image
But you can add another class to the non-dropdown buttons where you can over-ride this style if you need to. Alternatively, you could wrap the buttons in a div or something.

**Review by captainmangoC4 — APPROVED** [review]
